third_party/ibis/ibis_mssql/__init__.py

```diff
@@ -109,3 +114,74 @@ def list_primary_key_columns(self, database: str, table: str) -> list:
         with self.begin() as con:
             result = con.exec_driver_sql(list_pk_col_sql, parameters=(database, table))
             return [_[0] for _ in result.cursor.fetchall()]
+
+    LIST_DATABASE_SQL = """
```

Comment:
I'm interested in why we're declaring `LIST_DATABASE_SQL` as a class level attribute and not either a constant or a local variable inside `list_databases()`?

I'm not saying this is wrong, just wondered why.

Reply:
The only reason behind all your comments actually is the fact I followed the exact same approach already implemented for Teradata. For each comment I'll bring the snippet I followed:

https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/e75816b043cb63e9706e89c22d61776b2f821081/third_party/ibis/ibis_teradata/__init__.py#L76-L87

Reply:
Thanks for the context.

For this particular comment I was not saying it needed action, just trying to understand.

Anyway, I think being consistent with other methods in the same module is probably a good idea. Plus Teradata is slightly different because it is not subclassed from the same backend as some other engines, like SQL Server/Oracle/PostgreSQL.